platform-sdk/swirlds-platform-core/src/main/java/com/swirlds/platform/recovery/EventRecoveryWorkflow.java

```diff
@@ -381,7 +381,8 @@ private static ReservedSignedState handleNextRound(
         final PlatformState platformState = newState.getPlatformState();
 
         platformState.setRound(round.getRoundNum());
-        platformState.setRunningEventHash(getHashEventsCons(previousState.get().getHashEventsCons(), round));
+        platformState.setLegacyRunningEventHash(getHashEventsCons(
```

Comment:
Do you also need to set the new running hash here?

Reply:
I updated this to just set the null hash (i.e. `Cryptography.getNullHash())`). This recovery workflow already does not produce a state with the exact same hash as the original, so I feel no stress by making the result different in more ways.